editor/components/inserter/index.js

```diff
@@ -93,18 +88,17 @@ export default compose( [
 			getEditedPostAttribute,
 			getBlockInsertionPoint,
 			getSelectedBlock,
-			getSupportedBlocks,
 			getEditorSettings,
+			getInserterItems,
 		} = select( 'core/editor' );
-		const { allowedBlockTypes, templateLock } = getEditorSettings();
+		const { templateLock } = getEditorSettings();
```

Comment:
Minor: We can now remove the usage of template lock as getInserterItems if there is a lock getInserterItems will return no items.

Reply:
Fixed in 4f6e662b1 👌